docs/vectorizer-quick-start.md

```diff
@@ -38,7 +38,19 @@ On your local machine:
      data:
     ```
 
-1. **Start the services**
+1. **Start the database**
+   ```shell
+    docker compose up -d db
+    ```
+
+1. **Install pgai in your database**
+   ```shell
+   docker run --rm --entrypoint python \
+   timescale/pgai-vectorizer-worker:latest \
```

Comment:
Why don't you use the container created by the docker compose? Instead a `docker compose run` command using the `vectorizer-worker` service

Reply:
oh yeh that's a good idea! will change it